pkg/apis/psmdb/v1/psmdb_defaults.go

```diff
@@ -802,6 +803,30 @@ func (m *MultiAZ) reconcileAffinityOpts() {
 	}
 }
 
+func (m *MultiAZ) reconcileTopologySpreadConstraints(cr *PerconaServerMongoDB) {
+	if cr.CompareVersion("1.15.0") < 0 {
+		return
+	}
+
+	if len(m.TopologySpreadConstraints) == 0 {
```

Comment:
@pooknull  @egegunes  I am not sure that we need to add it by default. We need to use it only in case TopologySpreadConstraints is present in CR.

Reply:
agree